Add new independent service page for lifestyle

smailq opened the pull request:
Use `/services/curate-lifestyle` URL to see the page. It's not listed yet.

smailq commented:
Link to the page : https://curate-health-git-kyu-awa-50-create-new-420c90-fnhans-projects.vercel.app/services/curate-lifestyle

fnhan commented:
replace The Pillars of Lifestyle Medicine section with Program Timeline (swap the order)

fnhan commented:
1. Conditions We Address​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌​﻿​‍​﻿‍‌​﻿‍​​﻿​​​﻿​‍​﻿‌﻿​﻿‍​‌‍​‍​‍﻿‌​﻿‌​‌‍‌‌​﻿​﻿​﻿‌​​‍﻿‌​﻿‌​‌‍‌‌​﻿‌‍‌‍​‌​‍﻿‌​﻿‍‌​﻿​﻿​﻿‍​​﻿​​​‍﻿‌​﻿​​‌‍‌​​﻿‍‌​﻿​﻿‌‍​‍​﻿​‌‌‍​﻿‌‍​‌‌‍​‍‌‍‌​‌‍​‍‌‍‌‌​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‍‌‍﻿​‌‍﻿﻿‌‍​﻿‌‍‍﻿‌‌﻿﻿​﻿​﻿‌‌﻿﻿‌﻿‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌​﻿​‍​﻿‍‌​﻿‍​​﻿​​​﻿​‍​﻿‌﻿​﻿‍​‌‍​‍​‍﻿‌​﻿‌​‌‍‌‌​﻿​﻿​﻿‌​​‍﻿‌​﻿‌​‌‍‌‌​﻿‌‍‌‍​‌​‍﻿‌​﻿‍‌​﻿​﻿​﻿‍​​﻿​​​‍﻿‌​﻿​​‌‍‌​​﻿‍‌​﻿​﻿‌‍​‍​﻿​‌‌‍​﻿‌‍​‌‌‍​‍‌‍‌​‌‍​‍‌‍‌‌​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‍‌‍﻿​‌‍﻿﻿‌‍​﻿‌‍‍﻿‌‌﻿﻿​﻿​﻿‌‌﻿﻿‌﻿‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌ area, when it drops, can we have the entire section remain the same and not drop with the drop boxes? Maybe just accommodate extra padding there so the next section (Program Timeline doesn't keep shifting and moving every time a box drops with hover?)
2. Arrows for Program Timeline?
3. Additional Benefits: the photos have changed, check out the figma file. I got my designer to balance out all the photos so that the tint and hue/saturation is all matching. If you can follow that, that would be great. Can you also make sure I can easily replace each of these photos? 
4. Let's shift "How to join" section up and "Meet Our Team" section below that
5. "How to Join" section on mobile looks a bit funny with the spacing/padding of the photo and title "how to join"
6. Meet Our Team: should only be Dr. Eric, Dr. Lester, Dr. Gabriele, and myself. No one else should be there. If you can follow the design on Figma, that would be great. Eric should be the first, and larger. The drop down feature should push everything below it down.
7. The Pillars of Lifestyle Medicine: the photo and circles around the image has been redone on figma. If you can follow that design, that would be great!
8. FAQ and doctor's testimonial missing. Can you make the doctor's testimonials hidden for now, and I can toggle when I would like it to show?
9.

fnhan commented:
Curate Lifestyle Program detailed page:

1. Group Sessions​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍‌﻿‌﻿​‍‌‍﻿﻿‌﻿‌‌‌﻿​​‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍‌﻿‌﻿​‍‌‍﻿﻿‌﻿‌‌‌﻿​​‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌: can you make the photos change to colour with hover, on mobile, you can just keep it in colour
2. Need Assistance?​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‌‌﻿​﻿‌﻿​﻿‌‍‍‌‌﻿​﻿‌﻿‌​‌‍​‌‌‍﻿‍‌‍​﻿‌‍‌‌‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‌‌﻿​﻿‌﻿​﻿‌‍‍‌‌﻿​﻿‌﻿‌​‌‍​‌‌‍﻿‍‌‍​﻿‌‍‌‌‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌: are you able to reproduct the form? And have the form content stored somewhere?

smailq commented:
> Curate Lifestyle Program detailed page:
> 
> 1. Group Sessions​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍‌﻿‌﻿​‍‌‍﻿﻿‌﻿‌‌‌﻿​​‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍‌﻿‌﻿​‍‌‍﻿﻿‌﻿‌‌‌﻿​​‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌: can you make the photos change to colour with hover, on mobile, you can just keep it in colour

Done.

> 2. Need Assistance?​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‌‌﻿​﻿‌﻿​﻿‌‍‍‌‌﻿​﻿‌﻿‌​‌‍​‌‌‍﻿‍‌‍​﻿‌‍‌‌‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‌‌﻿​﻿‌﻿​﻿‌‍‍‌‌﻿​﻿‌﻿‌​‌‍​‌‌‍﻿‍‌‍​﻿‌‍‌‌‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌: are you able to reproduct the form? And have the form content stored somewhere?

Which service do you use again? Need to access that email list service to see if it's possible.

fnhan commented:
> > * Conditions We Address​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌​﻿​‍​﻿‍‌​﻿‍​​﻿​​​﻿​‍​﻿‌﻿​﻿‍​‌‍​‍​‍﻿‌​﻿‌​‌‍‌‌​﻿​﻿​﻿‌​​‍﻿‌​﻿‌​‌‍‌‌​﻿‌‍‌‍​‌​‍﻿‌​﻿‍‌​﻿​﻿​﻿‍​​﻿​​​‍﻿‌​﻿​​‌‍‌​​﻿‍‌​﻿​﻿‌‍​‍​﻿​‌‌‍​﻿‌‍​‌‌‍​‍‌‍‌​‌‍​‍‌‍‌‌​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‍‌‍﻿​‌‍﻿﻿‌‍​﻿‌‍‍﻿‌‌﻿﻿​﻿​﻿‌‌﻿﻿‌﻿‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌​﻿​‍​﻿‍‌​﻿‍​​﻿​​​﻿​‍​﻿‌﻿​﻿‍​‌‍​‍​‍﻿‌​﻿‌​‌‍‌‌​﻿​﻿​﻿‌​​‍﻿‌​﻿‌​‌‍‌‌​﻿‌‍‌‍​‌​‍﻿‌​﻿‍‌​﻿​﻿​﻿‍​​﻿​​​‍﻿‌​﻿​​‌‍‌​​﻿‍‌​﻿​﻿‌‍​‍​﻿​‌‌‍​﻿‌‍​‌‌‍​‍‌‍‌​‌‍​‍‌‍‌‌​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‍‌‍﻿​‌‍﻿﻿‌‍​﻿‌‍‍﻿‌‌﻿﻿​﻿​﻿‌‌﻿﻿‌﻿‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌ area, when it drops, can we have the entire section remain the same and not drop with the drop boxes? Maybe just accommodate extra padding there so the next section (Program Timeline doesn't keep shifting and moving every time a box drops with hover?)
> 
> Hm, the boxes are very long - to accommodate extra padding the section has to be too big. **I'm thinking a bit of extra padding, maybe just 1" (how ever many pixels that usually is), and having the lowest drop down drop almost to touch that new boarder. If we increase the padding here, let's make it symmetrical at the top of the section as well. Can you also make the drop down slower?** 
> 
> > * Arrows for Program Timeline? 
> 
> These are hard to do... I can try them but for me it'll take a long time. **Maybe make this a photo, and have it change with different screen adaptive dimensions?**
> 
> > * Additional Benefits: the photos have changed, check out the figma file. I got my designer to balance out all the photos so that the tint and hue/saturation is all matching. If you can follow that, that would be great. Can you also make sure I can easily replace each of these photos?
> 
> Yes, the photos are in Sanity, you can change them anytime.
> 
> Hm, I can add tint values to Sanity so you can customize each image. The original image files in the Figma does not have any tint, the designer changed tint % on top of each image on Figma.
> 
> > * Let's shift "How to join" section up and "Meet Our Team" section below that
> 
> Done
> 
> > * "How to Join" section on mobile looks a bit funny with the spacing/padding of the photo and title "how to join"
> 
> Done. Hide photo on mobile, it doesn't really fit.
> 
> > * Meet Our Team: should only be Dr. Eric, Dr. Lester, Dr. Gabriele, and myself. No one else should be there. If you can follow the design on Figma, that would be great. Eric should be the first, and larger. The drop down feature should push everything below it down.
> 
> Team section updated.
> 
> If it push everything below it down, then the whole page needs to move, like "Conditions" page. And these boxes' contents are long, can't have super long empty space, it'll look weird. **maybe have the Learn More with an arrow to the right, and have it link to the Our Team page with that particular practitioner profile already dropped down**
> 
> > * The Pillars of Lifestyle Medicine: the photo and circles around the image has been redone on Figma. If you can follow that design, that would be great!
> 
> Done.
> 
> > * FAQ and doctor's testimonial missing. Can you make the doctor's testimonials hidden for now, and I can toggle when I would like it to show?
> 
> My mistake - FAQ is added back.
> 
> Yes, I'll make this toggle.

fnhan commented:
> > Curate Lifestyle Program detailed page:
> > 
> > 1. Group Sessions​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍‌﻿‌﻿​‍‌‍﻿﻿‌﻿‌‌‌﻿​​‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍‌﻿‌﻿​‍‌‍﻿﻿‌﻿‌‌‌﻿​​‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌: can you make the photos change to colour with hover, on mobile, you can just keep it in colour
> 
> Done.
> 
> > 2. Need Assistance?​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‌‌﻿​﻿‌﻿​﻿‌‍‍‌‌﻿​﻿‌﻿‌​‌‍​‌‌‍﻿‍‌‍​﻿‌‍‌‌‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​﻿‌﻿‌​‌﻿‌‌‌‍‌​‌‍‍‌‌‍﻿﻿​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌‌‍‌‌​﻿​﻿‌‍‌​​﻿‌‍​﻿‌​​﻿‍​​﻿​﻿​﻿​﻿​‍﻿‌​﻿‌‍​﻿​​‌‍​‍​﻿​‌​‍﻿‌​﻿‌​‌‍‌‍​﻿‌‌​﻿‌​​‍﻿‌‌‍​‍‌‍​‍‌‍‌​‌‍​‍​‍﻿‌‌‍​‍‌‍‌​‌‍​‌​﻿‌﻿‌‍​﻿‌‍‌‌​﻿‍‌​﻿​‌​﻿​​​﻿‌‍‌‍‌‍‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‌‌‌﻿​‍‌﻿‌‍‌‍‍‌‌‍​﻿‌‍‌‌‌​﻿​‌‍‍‌‌‍‌‍‌‍‌‌‌﻿​﻿‌﻿‌​‌﻿‍‌‌‍﻿​‌‍‌‌‌‌​​‌﻿​‍‌‍﻿﻿‌‍‌﻿‌﻿​‍‌‍​‌‌‍﻿‌​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‌‌﻿​﻿‌﻿​﻿‌‍‍‌‌﻿​﻿‌﻿‌​‌‍​‌‌‍﻿‍‌‍​﻿‌‍‌‌‌‌​﻿‌‍‌‌‌‍​﻿‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍‌‌‌​‌‍‍‌‌﻿‌​‌‍﻿​‌‍‌‌​‍​‍‌﻿﻿‌: are you able to reproduct the form? And have the form content stored somewhere?
> 
> Which service do you use again? Need to access that email list service to see if it's possible. **I've been using google sheets when they actually fill out the quiz in the footer. However, can you just make it so it sends us an email with the info people fill in?**

fnhan commented:
<img width="1497" height="1032" alt="Screenshot 2025-08-07 214544" src="https://github.com/user-attachments/assets/7a760eba-70c5-490c-a81d-6cd1506465da" />
 
Lets try centering this profile, and make it slightly bigger than what it is now.

fnhan commented:
I notice The Pillars of Lifestyle Medicine circle person shifts a bit when you hover over the different circles. Can we fix this?

smailq commented:
> <img alt="Screenshot 2025-08-07 214227" width="1342" height="695" src="https://private-user-images.githubusercontent.com/151864662/475829848-1086bd18-e9eb-4b15-bbc2-a49a5efe38fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQ2NTU3NzAsIm5iZiI6MTc1NDY1NTQ3MCwicGF0aCI6Ii8xNTE4NjQ2NjIvNDc1ODI5ODQ4LTEwODZiZDE4LWU5ZWItNGIxNS1iYmMyLWE0OWE1ZWZlMzhmZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwOFQxMjE3NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZmY4Njc4OTFhN2MxODk2M2JlMjI5ZGVjY2Q4ZDk0MGFlMDc0OWYzOWJmMTQ0NjMwNzBkMmUwNTRiODlhMzgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Fp4bHdm3IvZxzZ9T2DeEkukYTqDsF_4KAXqrFDSpW7c"> Not aligned

Hm, I fixed this before already - try refreshing the page?  I tried to open and I can't reproduce.

Fixed the shifting of the pillar.

fnhan commented:
> > <img alt="Screenshot 2025-08-07 214227" width="1342" height="695" src="https://private-user-images.githubusercontent.com/151864662/475829848-1086bd18-e9eb-4b15-bbc2-a49a5efe38fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQ2NTU3NzAsIm5iZiI6MTc1NDY1NTQ3MCwicGF0aCI6Ii8xNTE4NjQ2NjIvNDc1ODI5ODQ4LTEwODZiZDE4LWU5ZWItNGIxNS1iYmMyLWE0OWE1ZWZlMzhmZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwOFQxMjE3NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZmY4Njc4OTFhN2MxODk2M2JlMjI5ZGVjY2Q4ZDk0MGFlMDc0OWYzOWJmMTQ0NjMwNzBkMmUwNTRiODlhMzgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Fp4bHdm3IvZxzZ9T2DeEkukYTqDsF_4KAXqrFDSpW7c"> Not aligned
> 
> Hm, I fixed this before already - try refreshing the page? I tried to open and I can't reproduce.
> 
> Fixed the shifting of the pillar.

I still see the shift in Name box, even with hard refresh and trying it on different computers. maybe it's a firefox thing? Can you take out the top "Name" label and have it just say Full Name in the text box?

The pillar person blinks every time you hover on different circle. Is there a way to take away the blinking?

smailq commented:
All done!

fnhan commented:
Outstanding:

- Conditions we address mobile is overlapping on next section
- Pillars arrangement around man
